frontend/js/restqueries.js

```diff
@@ -27,6 +27,29 @@ function retreiveFromDB(collection, query, populateCallback) {
 	}
 }
 
+function sendCommandToDB(data, populateCallback) {
```

Comment:
Nice descriptive name, and reusable function. Only change I recommend would be changing data to something like command, since your function name mentions sending a command, but it doesn't take a parameter related to command.